packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -90,21 +99,78 @@ const inlineMobileStyles = ({
     ${getInlineStyles({ theme })}
   `;
 
+const actionStyles = ({ theme }: StyleProps) => css`
```

Comment:
Last thing: Figma says that when they're aligned horizontally (desktop), buttons in the group should have the same width. So fo example a "Cancel this action" and an "OK" button should have the same width even if the strings have wildly different lengths.

I think we could do it with `display: inline-grid;`: https://codepen.io/robinmetral/pen/vYZZMvw?editors=1100

Reply:
Neat idea, unfortunately, CSS grid is not possible under our current browser support policy 😕

Reply:
Frankly, I don't think this is helpful behavior since it can lead to problems with long strings. Can we challenge this?

Reply:
I think we can—or at least keep the status quo until we can support inline grid without polyfills

Reply:
@amelako for your notes to discuss tomorrow with Giao-Chung 🙂